src/components/common/ColonyHome/ColonyHome.tsx

```diff
@@ -78,6 +78,8 @@ const ColonyHome = () => {
     });
   };
 
+  const searchParams = selectedTeam && `?team=${selectedTeam.nativeId}`;
```

Comment:
This is a little bit too manual for my taste. Please at least use the `URLSearchParams` API: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams